validator/src/main/java/com/google/daq/mqtt/registrar/DeviceExceptionManager.java

```diff
@@ -9,15 +9,15 @@
 import java.util.Optional;
 import java.util.regex.Pattern;
 
-public class DeviceExceptionManager {
+class DeviceExceptionManager {
```

Comment:
Is this a lint-based change?

Reply:
yes -- public classes require javadoc, and there's no need for this to be public since it's only used internally.